### ISSUE TITLE
First use issue - release-20180307

### ISSUE THREAD

**Issue by Bogop21** [OpenRA/OpenRA#15580]
Hello, 

I have installed "release-20180307", today. No apparent issue during the installation. 
I have Windows 10 (x64). 

When I attempt to open any of the OpenRA games, a Fata Error message appears telling me to look at the Crash log or at the FAQ's. 
The Crash Log is empty. 

Let me know if anyone has encountered this situation 

Please clarify the installation and First Use process. 

Thanks,

**Comment by rautamiekka** [comment]
Smells like lack of, or insufficient, OpenGL. What graphics card are you running and the driver version ?

**Comment by pchote** [comment]
This is almost always caused by an antivirus/security tool preventing OpenRA from starting - any other crash should write out crash logs.

**Comment by rautamiekka** [comment]
Might be lack of .NET. The thing is the requirements are old and there's been mentionings of requiring one of the newest, so if you'll install .NET 4.7.x that's 1 thing less.

**Comment by matjaeck** [comment]
@Bogop21 could you give us an update on this? 

> This is almost always caused by an antivirus/security tool preventing OpenRA from starting

Do you have such software installed and did whitelisting help?

**Comment by pchote** [comment]
These anti-virus issues are really starting to annoy me, so reopening and adding to Next + 1.

We should be able to detect this based on the exit code and then show a specific error dialog warning about anti-virus conflicts.

**Comment by Mailaender** [comment]
Was the anti-virus solution Windows Defender?

**Comment by matjaeck** [comment]
Default Windows defender does not cause problems, however you have to get around the security warning when installing as the application is not signed. Windows 10 firewall prompts to allow network access when starting the game the first time. 

Tl;dr: This is IMO not caused by default Win10 defender/firewall but thirdparty stuff.

**Comment by pchote** [comment]
I'm struggling to find a way to reproduce this in my windows virtual machine. Does anyone know how to force avast to block OpenRA?

**Comment by pchote** [comment]
I can't progress this without help, so dropping from the milestone.

**Comment by abcdefg30** [comment]
Closing because of inactivity. If we want to keep a ticket open because of antivirus problems we should open a properly named ticket with concise information.
